docs/docs/backup_howto.md

```diff
@@ -0,0 +1,205 @@
+# Backup strategy guide
+
+This guide explains how to design and implement a backup strategy in Xen Orchestra.  
+Instead of a simple list of questions and answers, it walks you through **key decisions** and **best practices** before, during, and after setting up backups.
+
+---
+
+## Glossary of backup types & features
+
+- **Full backup** – Copies the entire VM each time, regardless of previous backups.  
+- **Delta backup** – Saves only the changes since the last backup, reducing storage needs.  
+- **Incremental replication** – Formerly called Continuous Replication (CR). Keeps a near-real-time copy of a VM on another host or pool.  
+- **Full replication** – Formerly called Disaster Recovery (DR). Creates a complete replica of a VM on another host or pool.  
+- **Mirror backup** – Keeps an exact mirror of the VM backup set, with no retention—only the latest copy is kept.  
+- **Sequence** – Groups multiple backups or replications in a specific order to be run as a single job.  
+- **Long-term retention** – Keeps backups over extended periods (weeks, months, or years) for compliance or archival purposes.  
+- **Remote** – A storage location for backups (e.g., NFS, SMB, S3).  
+- **File restore** – Allows restoring individual files from a VM backup without restoring the full VM.
+
+---
+
+## 1. What should I do before setting up my backup?
+
+Before creating your first backup job in Xen Orchestra, consider the following:
+
+### Remote
+Choose and configure your remote storage.  
+- Supported types: NFS, SMB/CIFS, S3-compatible object storage, XOSAN, or local storage.  
+- Ensure proper permissions and network connectivity.  
+- Test write/read performance before relying on it.
+
```

Comment:
we should probably have something about block storage encryption and immutability here.